src/components/used-storage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Progress } from 'antd';
+import PropTypes from 'prop-types';
+
+function convertBytesToHigherUnit(bytes) {
+  const kilobytes = bytes / 1000;
+  if (kilobytes < 1000) {
+    return `${kilobytes} KB`;
+  }
+  const megabytes = kilobytes / 1000;
+  if (megabytes < 1000) {
+    return `${megabytes} MB`;
+  }
+  const gigabytes = megabytes / 1000;
+  return `${gigabytes} GB`;
+}
```

Comment:
We have the module `pretty-bytes` which does "pretty" much this (it's used for example in the CDN file browser).

Reply:
I was under the impression this is the only case where we show byte size to the user and didn't want to introduce a library for that.
Cool, used `pretty-bytes` 👍